packages/widget/src/index.js

```diff
@@ -75,11 +75,13 @@ export default {
         };
 
         param.addEventListener('value', (value) => {
-          params[name] = {
-            value,
-            ...(isSelection(param) ? { predicate: String(param.predicate()) } : {}),
-          };
-          view.model.set('params', params);
+          view.model.set('params', {
+            ...params,
```

Comment:
Hmm, how is this correct? This doesn't update the `params` variable so doesn't that mean we would lose any changes to params that are not `name`?

Reply:
Maybe this would be correct if we instead got the params from `view.model.get('params')` before setting them again.

Reply:
Sorry I far too shallow tested this.
When I get from `view.model.get('params')` it works but there are still issues: I only see `range` and `click` entries and `click` has strange defaults
[Screencast from 2025-04-19 19-51-06.webm](https://github.com/user-attachments/assets/ca25bc7b-256d-46e9-860a-750c29106fbd)

Reply:
Whereas without my changes there are more entries for `domain` and `colors` - I presume non-selection params that should be maintained:
![image](https://github.com/user-attachments/assets/e601eb65-1e77-4e62-b6e2-8dc7235ea488)

Reply:
Above I made a different error in the initial `model.set`.
Starting from `view.model.get('params')` as suggested does work fine in this example.
[Screencast from 2025-04-19 20-04-17.webm](https://github.com/user-attachments/assets/20df5593-c26e-4e5f-a301-abde2af31f81)